recipes/indi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "indi" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/indilib/indi/archive/v{{ version }}.tar.gz
+  sha256: 82004db3d1e45f8852c1b871fdb925a9ce0afda4f8f92de0a197b9b868933abf
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - cmake
+  host:
+    - cfitsio
+    - gsl
+    - zlib
+
+test:
+  commands:
+    - test -x $PREFIX/bin/indiserver  # [unix]
+    - test -d $PREFIX/include/libindi/  # [unix]
+    - test -f $PREFIX/lib64/libindiclient.a  # [unix and linux64]
+    - test -f $PREFIX/lib/libindiclient.a  # [unix and not linux64]
```

Comment:
You should be building shared libraried if you can

Reply:
I looked through their CMakeLists and don't think this lib can be built shared without changes upstream

Reply:
I think if you set this variable at the command line, you'll get them: https://cmake.org/cmake/help/v3.0/variable/BUILD_SHARED_LIBS.html